gstreamer/src/tags.rs

```diff
@@ -39,7 +39,7 @@ macro_rules! impl_tag(
         }
 
         lazy_static! {
-            static ref $rust_tag: &'static str =
+            pub static ref $rust_tag: &'static str =
```

Comment:
This should probably be `pub(crate)`?

Reply:
It could. Note that this was part of #119 though. So we might want to address it as a separate PR (I can do it tonight).

Reply:
OK, then let's do that :)